units/host_drawdown.go

```diff
@@ -158,7 +158,7 @@ func (j *hostDrawdownJob) checkAndTerminateHost(ctx context.Context, h *host.Hos
 		(*drawdownTarget)--
 		j.Terminated++
 		j.TerminatedHosts = append(j.TerminatedHosts, h.Id)
-		if err = h.SetDecommissioned(ctx, evergreen.User, true, "host decommissioned due to overallocation"); err != nil {
+		if err = h.SetDecommissioned(ctx, evergreen.User, true, true, "host decommissioned due to overallocation"); err != nil {
```

Comment:
I believe this should actually be false for `terminateIfBusy` because the host drawdown job is performing a function similar to the idle host job (it's trying to gracefully turn off hosts).

Reply:
updated (it makes sense because checkTaskGroup was true)